virtualhome/src/groovy/aaf/vhr/email_password_reset_without_phone.gsp

```diff
@@ -0,0 +1,37 @@
+Hello ${managedSubject.cn.encodeAsHTML()},<br><br>
+
+You are attempting to recover a lost password.<br><br>
+
+Please note that you do not have a mobile number configured for this account.<br>
+
+A mobile number is required for SMS confirmation of your identity.<br>
+
+<strong>Please contact one of your listed administrators to obtain the required SMS code!</strong><br><br>
+
+<g:if test="${groupAdminRole.subjects?.size() > 0}">
+  <h5>Primary Administrators</h5>
+  <ul>
+    <g:each in="${groupAdminRole.subjects?.sort{it.cn}}" var="subject">
+      <li><g:fieldValue bean="${subject}" field="cn"/> - <a href="mailto:${subject.email}"><g:fieldValue bean="${subject}" field="email"/></a></li>
+    </g:each>
+  </ul>
+</g:if>
+
+<g:if test="${organizationAdminRole.subjects?.size() > 0}">
+<h5>Secondary Administrators</h5>
+  <ul>
+    <g:each in="${organizationAdminRole.subjects?.sort{it.cn}}" var="subject">
+      <li><g:fieldValue bean="${subject}" field="cn"/> - <a href="mailto:${subject.email}"><g:fieldValue bean="${subject}" field="email"/></a></li>
+    </g:each>
+  </ul>
+</g:if>
+
+Once you have obtained the SMS code from your administrator, <a href="${emailURL}">click the link</a> to proceed with your password reset.<br><br>
+
+<g:if test="${groupAdminRole.subjects?.size() > 0}">
```

Comment:
I think the condition may have to be an "or" over organizationAdminRole and groupAdminRole

Reply:
Check the latest commit

Reply:
Great, thanks!